catalogs/sources/gtfs/schedule/de-unknown-postauto-ag-gtfs-2053.json

```diff
@@ -0,0 +1,23 @@
+{
+    "mdb_source_id": 2053,
+    "data_type": "gtfs",
+    "provider": "PostAuto AG",
```

Comment:
@AlfredNwolisa Where did this info come from? The country code should be CH

Reply:
Got it from the agency.txt file.

Reply:
makes sense - I imagine there's multiple agencies included? Let's use "Swiss Federal Railways (SBB)" since that's where the data comes from

Reply:
yes, there were multiple agencies. okay i will update it to ssb